rioxarray/rioxarray.py

```diff
@@ -871,6 +885,8 @@ def reproject(
         resolution: float or tuple(float, float), optional
             Size of a destination pixel in destination projection units
             (e.g. degrees or metres).
+        shape: tuple(int, int), optional
+            Shape of the destination in pixels. Cannot be used together with resolution.
```

Comment:
Mind adding a note that the order in the tuple is (dst_height, dst_width)?

Reply:
:+1: